packages/node_modules/@ciscospark/react-component-video/src/index.js

```diff
@@ -3,13 +3,19 @@ import classNames from 'classnames';
 
 import styles from './styles.css';
 
-export default function Video({src}) {
+export default function Video({src, audio}) {
   // Need autoPlay to start the video automagically
   return (
-    <video autoPlay className={classNames(styles.video)} src={src} />
+    <video
+      autoPlay
+      className={classNames(styles.video)}
+      muted={audio}
```

Comment:
maybe `audioMuted` instead of just `audio`? i would expect `audio === true` to produce audio, not mute audio.